src/pages/Account/AdminEvent.tsx

```diff
@@ -154,39 +143,18 @@ const AdminEvent: React.FunctionComponent<PropsType> = ({
             )}
           </div>
           <div className="input-group">
-            <label>End Time</label>
+            <label htmlFor="duration_hours">Duration (hours)</label>
             <input
-              type="date"
```

Comment:
Is this for burningman? I thought we were going to (re)create a Schedule, completely separate from Events, more like what we currently have on the PartyMap.